object/init.go

```diff
@@ -62,9 +62,12 @@ func initBuiltInApplication() {
 		CreatedTime:    util.GetCurrentTime(),
 		DisplayName:    "Casdoor",
 		Logo:           "https://cdn.casbin.com/logo/logo_1024x256.png",
+		HomepageUrl:    "https://casdoor.org",
```

Comment:
Why do you commit something that is already there?

https://github.com/casbin/casdoor/blob/0f8a46fe0bd286caa0aed7658dff22c8a67e5927/object/init.go#L53-L73